### PR TITLE
improvement(results): add sending text and screenshots

### PR DESCRIPTION
**Pull request by soyacz**
improvement(results): add sending text and screenshots

Implemented support for sending text results to argus.
Text results are ignored in graphs.
If a text result is a link to image it is shown as button which opens
given image in a modal (e.g. screenshots). This enables to send
screenshots from latency_decorator in SCT.

Also did a bit of refactor around results (moved fetching result tables
to `ResultsService` where fits better.

improvement(screenshots): move to separate component

Moved screenshot modal to separate component to reuse it.
Also added 'close by esc' feature.

### CODE REVIEW

**Comment by soyacz** [comment]
This is how it will look in results:
image
Clicking button opens image in modal - same way as in 'Screenshots' tab.

**Comment by fruch** [comment]
> This is how it will look in results: image Clicking button opens image in modal - same way as in 'Screenshots' tab.

how the copy to markdown would look ?

**Comment by soyacz** [comment]
> how the copy to markdown would look ?

|  | P90 read [ms] | P99 read [ms] | duration [HH:MM:SS] | Overview | QA dashboard |
| --- | --- | --- | --- | --- | --- |
| Cycle #&#8203;1 | $${\color{red}5.03}$$ | $${\color{green}9.72}$$ | $${\color{green}00:28:35}$$ | view | view |

File: frontend/TestRun/ResultsTab.svelte [hunk 1]
```diff
@@ -337,9 +318,11 @@
                                 <tr>
                                     <td>{row}</td>
                                     {#each result.columns as col}
-                                        <td class="{ResultCellStatusStyleMap[result.table_data[row][col.name]?.status || 'NULL']}">
-                                            {formatValue(result.table_data[row][col.name]?.value, result.table_data[row][col.name]?.type || "NULL")}
-                                        </td>
+                                        {#key result.table_data[row][col.name]}
+                                            <td class="{ResultCellStatusStyleMap[result.table_data[row][col.name]?.status || 'NULL']}">
+                                                <Cell cell={result.table_data[row][col.name]} bind:selectedScreenshot/>
```
Comment (k0machi) [inline frontend/TestRun/ResultsTab.svelte:323]: This component seems to be missing from PR.

**Review by k0machi — CHANGES_REQUESTED** [review]

File: frontend/TestRun/ResultsTab.svelte (hunk 1 above)
Comment (soyacz) [reply frontend/TestRun/ResultsTab.svelte:323]: true, updated.

**Review by k0machi — APPROVED** [review]

**Pull request merged**